src/main/java/tests/CheckboxTest.java

```diff
@@ -0,0 +1,44 @@
+package tests;
+import org.testng.annotations.Test;
+import pages.CheckboxPage;
+import pages.ElementsPage;
+import pages.HomePage;
+import pages.TextboxPage;
+
+import static common.TestBase.driver;
+
+public class CheckboxTest extends TestCase {
+    @Test
+    public void testCheckbox() {
+        HomePage homePage = new HomePage(testBase.driver);
+        ElementsPage elementsPage = new ElementsPage(testBase.driver);
+        CheckboxPage checkboxPage = new CheckboxPage(testBase.driver);
+
+        homePage.clickElements();
+        elementsPage.clickElement("Check Box");
+        checkboxPage.selectCheckbox("Home");
+        checkboxPage.clickOnExpandIcon("Home");
+        checkboxPage.verifyTextDisplayed("You have selected :\n" +
+                "home\n" +
+                "desktop\n" +
+                "notes\n" +
+                "commands\n" +
```

Comment:
String expectedText ="You have selected : home desktop notes commands documents workspace react angular veu office public private classified general downloads wordFile excelFile"
assertEquals(actualText, expectedText );